MapboxNavigation/Error.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+
+/**
+ Custom Error code key used in returned in `voiceController(_:didInterrupt:with:)`.
+ */
+public let SpokenInstructionErrorCodeKey = "SpokenInstructionErrorCodeKey"
```

Comment:
A global constant that’s accessible to Objective-C needs to use the class prefix `MB` in both its name and value. Additionally, the word “Key” should only appear in the name of the constant, not its value.